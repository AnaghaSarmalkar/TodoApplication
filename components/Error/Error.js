import React from "react";
import { Text, View } from "react-native";

export default function Error({ message }) {
  const errorText = `Error getting resource: ${message}`;
  return (
    <View>
      <Text>{errorText}</Text>
    </View>
  );
}
