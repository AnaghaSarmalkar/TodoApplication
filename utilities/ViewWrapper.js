import React from "react";
import { View } from "react-native";

export default function ViewWrapper(props) {
  return (
    <View
      {...props}
      // style={{
      //   flex: 1,
      //   alignItems: "center",
      //   justifyContent: "center",
      //   // paddingTop: insets.top,
      // }}
    >
      {props.children}
    </View>
  );
}
