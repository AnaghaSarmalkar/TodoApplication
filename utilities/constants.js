import { Platform } from "react-native";

export const isIos = Platform.OS === "ios";
export const isWeb = Platform.OS === "web";
export const isAndroid = Platform.OS === "android";
