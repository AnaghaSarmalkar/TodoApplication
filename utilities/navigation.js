export function setPageTitle(navigation, title) {
  if (title) {
    navigation.setOptions({
      title: title,
    });
  }
}
