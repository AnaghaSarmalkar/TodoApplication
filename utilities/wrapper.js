import React from "react";
import { SafeAreaView } from "react-native-safe-area-context";

function SafeAreaWrapper({ children }) {
  <SafeAreaView>{children}</SafeAreaView>;
}
