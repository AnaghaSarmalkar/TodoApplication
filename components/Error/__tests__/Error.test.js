import React from "react";
import { shallow } from "../../../utilities/testing";
import Error from "../Error";

describe("", () => {
  it("renders Error component", async () => {
    const errorMessage = "Test Error Message";
    const wrapper = shallow(<Error message={errorMessage} />);

    expect(wrapper).toMatchSnapshot();
  });
});
