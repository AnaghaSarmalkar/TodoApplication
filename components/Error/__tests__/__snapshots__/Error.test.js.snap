// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` renders Error component 1`] = `
<View>
  <Text>
    Error getting resource: Test Error Message
  </Text>
</View>
`;
